IPython/html/static/edit/js/notificationarea.js

```diff
@@ -17,10 +17,13 @@ define([
      */
     EditorNotificationArea.prototype.init_notification_widgets = function () {
         var that = this;
-        var enw = this.new_notification_widget('editor');
+        var savew = this.new_notification_widget('save');
         
-        this.events.on("save_succeeded.TextEditor", function() {
-            enw.set_message("File saved", 2000);
+        this.events.on("file_saving.Editor", function() {
+            savew.set_message("Saving File...");
```

Comment:
Do we really need a notification for this? It seems more likely to cause an annoying flicker than to ever be useful.